dynamodb/dynamodb.sim.w

```diff
@@ -0,0 +1,214 @@
+bring sim;
+bring util;
+bring cloud;
+bring "./dynamodb-types.w" as dynamodb_types;
+bring "./dynamodb-client.w" as dynamodb_client;
+
+interface Process {
+  inflight kill(): void;
+}
+
+struct SpawnOptions {
```

Comment:
@Chriscbr any idea when this will be supported in the sdk? This is the 4th implementation I've seen of this code (and all of them have issues I am sure)